src/List.svelte

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+  import Loading from "@app/Loading.svelte";
+
+  export let items: any[];
+  export let query: any;
+
+  // Being passed to more slot to handle loading state of trigger
```

Comment:
Can't parse this sentence

Reply:
sounded clear to me at the moment 🤣 
Will rephrase it